.ci-support/merge-install-production-branch.sh

```diff
@@ -12,28 +12,65 @@ set -x
 #
 # PRODUCTION_BRANCH = The production branch (default=production)
 # PRODUCTION_FORK = The production fork (default=illinois-ceesd)
-#
-MIRGE_HOME=${1:-"."}
-PRODUCTION_BRANCH=${PRODUCTION_BRANCH:-"production"}
-PRODUCTION_FORK=${PRODUCTION_FORK:-"illinois-ceesd"}
 
+MIRGE_HOME=${1:-"."}
 echo "MIRGE_HOME=${MIRGE_HOME}"
+
+if [[ "$GITHUB_HEAD_REF" == "$PR_MAIN_BRANCH" ]]; then
+  # GITHUB_HEAD_REF is only set for PR builds.
+  # https://docs.github.com/en/actions/learn-github-actions/environment-variables
+  PRODUCTION_BRANCH="production-merge/${PR_MAIN_BRANCH}"
+  PRODUCTION_FORK=${PR_PRODUCTION_FORK:-"illinois-ceesd"}
+else
+  PRODUCTION_BRANCH="${PRODUCTION_BRANCH:-"production"}"
+  PRODUCTION_FORK=${PRODUCTION_FORK:-"illinois-ceesd"}
+fi
+
+
 echo "PRODUCTION_FORK=$PRODUCTION_FORK"
 echo "PRODUCTION_BRANCH=$PRODUCTION_BRANCH"
 
-cd ${MIRGE_HOME}
+cd "${MIRGE_HOME}" || exit 1
 git status
 
-# This junk is needed to be able to execute git commands properly
-git config user.email "ci-runner@ci.machine.com"
-git config user.name "CI Runner"
+if [[ "$CI" == "true" ]]; then
+  # This is needed in order for git to create a merge commit
```

Comment:
Could mention here where $CI is set

Reply:
https://github.com/illinois-ceesd/mirgecom/compare/5e3ce201e251de56888e9fbe33b50def1ff8ee6a..e3725cd0d5cfb267c7deb3b0906977a0c38d5bf9